package exception;

public class ExceptionArray {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
	
	try {	int arr[]= {2,4,6};
		System.out.println(arr[9]);
	}
	
catch(Exception a){
	System.out.println(a);
}
}
}
